packages/components/Menu/src/MenuTrigger/getRevisedProps.ts

```diff
@@ -0,0 +1,48 @@
+import { AccessibilityActionEvent } from 'react-native';
+import { memoize } from '@fluentui-react-native/framework';
+import { InteractionEvent, isMouseEvent } from '@fluentui-react-native/interactive-hooks';
+import { MenuTriggerChildProps, MenuTriggerState } from './MenuTrigger.types';
+
+export const getRevisedProps = memoize(getRevisedPropsWorker);
+function getRevisedPropsWorker(state: MenuTriggerState, props: any): MenuTriggerChildProps {
+  const revisedProps = { ...state.props };
+  if (props.accessibilityState) {
+    revisedProps.accessibilityState = { ...state.props.accessibilityState, ...props.accessibilityState };
+  }
+
+  if (props.accessibilityActions) {
+    revisedProps.accessibilityActions = [...state.props.accessibilityActions, ...props.accessibilityActions];
+  }
+
+  if (props.onAccessibilityAction) {
+    revisedProps.onAccessibilityAction = (e: AccessibilityActionEvent) => {
+      props.onAccessibilityAction(e);
+      state.props.onAccessibilityAction(e);
+    };
+  }
+
+  if (props.onClick) {
+    revisedProps.onClick = (e: InteractionEvent) => {
+      props.onClick(e);
+      state.props.onClick(e);
```

Comment:
Not sure it does, but does order matter? I've normally seen our "injected logic' fire first before the client passed callback. I think this is the opposite?

Reply:
I don't think it really matters, I picked this order because that's what web has. I guess if there's a way for the client to tell us to not run our version of the event by telling us to not propagate the event or something, that would be a reason to run the client version first

Reply:
But I can switch it for consistency for now